test/integration/narrow/routes/reset-payment-request.test.js

```diff
@@ -0,0 +1,70 @@
+jest.mock('../../../../app/plugins/crumb')
+jest.mock('../../../../app/api')
+const { post } = require('../../../../app/api')
+jest.mock('../../../../app/auth')
+const createServer = require('../../../../app/server')
+const { schemeAdmin } = require('../../../../app/auth/permissions')
+const Boom = require('@hapi/boom')
+
+const url = '/payment-request/reset'
+const validInvoiceNumber = 'S1234567S123456V001'
+let server
+let auth
+
+describe('Reset payment request', () => {
+  beforeEach(async () => {
+    auth = { strategy: 'session-auth', credentials: { scope: [schemeAdmin] } }
+    jest.clearAllMocks()
+    server = await createServer()
+  })
+
+  afterEach(async () => {
+    await server.stop()
+  })
+
+  test('returns 403 and no permission', async () => {
+    auth.credentials.scope = []
+    const res = await server.inject({ method: 'GET', url, auth })
+    expect(res.statusCode).toBe(403)
+  })
+
+  test('returns 302 no auth', async () => {
```

Comment:
Test name is unclear.

Does this mean `when no authorisation is provided`?

Reply:
Updated